llvm/lib/Target/RISCV/RISCVInstrInfoVPseudos.td

```diff
@@ -6159,7 +6167,7 @@ defm PseudoVLSEG : VPseudoUSSegLoadFF;
 //===----------------------------------------------------------------------===//
 // 11.1. Vector Single-Width Integer Add and Subtract
 //===----------------------------------------------------------------------===//
-defm PseudoVADD   : VPseudoVALU_VV_VX_VI;
+defm PseudoVADD   : VPseudoVALU_VV_VX_VI<Commutable=1>;
```

Comment:
Would it be easier to just wrap the pseudos that are commutable in `let isCommutable=1` rather than threading it through the classes?

Reply:
For most RVV pseudos, only `_VV` variants can be commutable, so we can't simply wrap these `defm`s in `let isCommutable=1`.